test/integration/recipe.test.js

```diff
@@ -33,22 +34,7 @@ describe('Recipes', function () {
       await jwksMock.stop()
     })
 
-    beforeEach(async function () {
-      nock(`http://${IDENTITY_SERVICE_HOST}:${IDENTITY_SERVICE_PORT}`)
-        .get('/v1/members/foobar3000')
-        .reply(200, {
-          alias: 'foobar3000',
-          address: '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty',
-        })
-        .get('/v1/members/invalid')
-        .reply(404, {})
-        .get('/v1/members/error')
-        .reply(500, {})
-    })
-
-    afterEach(async function () {
-      nock.cleanAll()
-    })
+    setupIdentityMock()
```

Comment:
it kinda defeats the integration e2e being e2e. Anyway, we have made a head start with this approach. I think at some point once we have covered all the endpoints. We can start thinking of spinning up all services (github actions/flows) and running these tests without needing to mock or anything. It also could be something like a postman collection that could be exported.

I think we should start the service before running these so express app does not need to be initiated this way.

Reply:
Depends on what we mean by integration. At this point you could call these service tests maybe? We will indeed very similar e2e integration tests, but as you said the goal here is to keep us moving whilst we have auth issues